src/main/java/com/mcitmocks/mcitmocks/MCITMocksApplication.java

```diff
@@ -18,5 +22,11 @@ String test () {
     public static void main(String[] args) {
         SpringApplication.run(MCITMocksApplication.class, args);
     }
-
+    @Bean
+    CommandLineRunner commandLineRunner(UserRepository userRepository){
+        return args -> {
+            User mario= new User("mario@seas.upenn.edu","cst");
```

Comment:
Just one note - TimeZone will be in standard format like `America/Chicago`

Reply:
got it! Thanks